<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startSearchButton">
        <property name="text">
         <string>Start Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stopSearchButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="markingButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteFilesButton">
        <property name="text">
         <string>Delete Files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>320</height>
         </size>
        </property>
        <property name="title">
         <string>Compare By</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButton_6">
           <property name="text">
            <string>Rad&amp;ioButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="sizeThenMd5RadioButton">
           <property name="text">
            <string>Size, then &amp;Md5</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="radioButton_5">
           <property name="text">
            <string>R&amp;adioButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_3">
           <property name="text">
            <string>&amp;Size</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>320</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="CommonListWidget" name="searchDirsWidget">
         <attribute name="title">
          <string>Search Locations</string>
         </attribute>
        </widget>
        <widget class="TFilterWidget" name="filtersWidget">
         <attribute name="title">
          <string>Filters</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="exclusionsWidget">
         <attribute name="title">
          <string>Exclusions</string>
         </attribute>
        </widget>
        <widget class="CommonListWidget" name="protectFoldersWidget">
         <attribute name="title">
          <string>Protect Folders</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="resultPreviewTab">
         <attribute name="title">
          <string>Preview</string>
         </attribute>
         <layout class="QGridLayout" name="resultPreviewTabLayout"/>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="operationsTabsWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="operationsResultTab">
       <attribute name="title">
        <string>Result List</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="SearchRessultTableWidget" name="tableWidget">
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Group</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>File Size Byte</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>File Size</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>File Name</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ext</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Create Date</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Modifed Date</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>FileHash</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Full Path</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="operationProcessingTab">
       <attribute name="title">
        <string>Progress</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="labelStartDate">
          <property name="text">
           <string>labelStartDate #############################################</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelExecutedTime">
          <property name="text">
           <string>labelExecutedTime ##########################################</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelAproximateTime">
          <property name="text">
           <string>labelAproximateTime ########################################</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEndDate">
          <property name="text">
           <string>labelEndDate ##############################################</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="progressLabel">
          <property name="text">
           <string>progressLabel ############################################# </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentOperationLabel">
          <property name="text">
           <string>currentOperationLabel #######################################</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBarFileSize">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressbarFiles">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actio&amp;ns</string>
    </property>
    <widget class="QMenu" name="menuAction1">
     <property name="title">
      <string>&amp;Save Database</string>
     </property>
     <addaction name="actionSave_To_File"/>
     <addaction name="actionSave_To_Memory"/>
    </widget>
    <addaction name="menuAction1"/>
    <addaction name="actionInvalidate_Files_Hash_Database"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionSave_To_File">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Save To File</string>
   </property>
  </action>
  <action name="actionSave_To_Memory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save &amp;To Memory</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionClearSelection">
   <property name="text">
    <string>clearSelection</string>
   </property>
   <property name="toolTip">
    <string>clearSelection</string>
   </property>
  </action>
  <action name="actionInvertSelection">
   <property name="text">
    <string>invertSelection</string>
   </property>
   <property name="toolTip">
    <string>invertSelection</string>
   </property>
  </action>
  <action name="actionSelectByPath">
   <property name="text">
    <string>selectByPath</string>
   </property>
  </action>
  <action name="actionInvalidate_Files_Hash_Database">
   <property name="text">
    <string>&amp;Invalidate Files Hash Database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SearchRessultTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">components/searchressulttablewidget.h</header>
   <slots>
    <slot>deleteSelectedItems()</slot>
    <slot>unSelectItems()</slot>
    <slot>selectItemsByPath()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>CommonListWidget</class>
   <extends>QWidget</extends>
   <header location="global">components/commonlistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TFilterWidget</class>
   <extends>QWidget</extends>
   <header location="global">components/tfilterwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>deleteFilesButton</sender>
   <signal>clicked()</signal>
   <receiver>tableWidget</receiver>
   <slot>deleteSelectedItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectByPath</sender>
   <signal>triggered()</signal>
   <receiver>tableWidget</receiver>
   <slot>selectItemsByPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearSelection</sender>
   <signal>triggered()</signal>
   <receiver>tableWidget</receiver>
   <slot>unSelectItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
